pkgs/by-name/le/leftwm-config/package.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  pname = "leftwm-config";
+  version = "0-unstable-2024-03-13";
+
+  src = fetchFromGitHub {
+    owner = "leftwm";
+    repo = "leftwm-config";
+    rev = "a9f2f21ece3a01d6c36610295ae3163644d3f99e";
+    hash = "sha256-wyb/26EyNyBJeTDUvnMxlMiQjaCGBES8t4VteNY1I/A=";
+  };
+
+  cargoHash = "sha256-fiB8y4v1gc8d2gvqsgdQ4xHIjzrLxcqp8ybk09tOO8U";
+
+  meta = {
+    broken = (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
We should add a really short comment why those failed

Reply:
I think that's a leftover from the original commits i based the pr on, but I'm gonna run it on an aarch64 to confirm and/or get output logs

Reply:
Update: I checked above, I have definitely tested this before on leftwm-theme and removed it, but didn't see it was on config too. Gonna build to be sure, but it might be okay for removal

Reply:
Aaand it works! Removed